src/quotamodel.c

```diff
@@ -220,10 +220,10 @@ static void transfer_table_for_quota(int64 totalsize, QuotaType type, Oid *old_k
 
 /* functions to refresh disk quota model*/
 static void refresh_disk_quota_usage(bool is_init);
-static void calculate_table_disk_usage(bool is_init, HTAB *local_active_table_stat_map);
+static void calculate_table_disk_usage(bool is_init);
 static void flush_to_table_size(void);
 static bool flush_local_reject_map(void);
-static void dispatch_rejectmap(HTAB *local_active_table_stat_map);
+static void dispatch_rejectmap(const char *active_oids);
```

Comment:
Since dispatch_rejectmap doesn't take a map as an argument, have you considered renaming it?

Reply:
No! The word `rejectmap` in the name of the `dispatch_rejectmap` function does not refer to its argument, but to the shared hashmap `disk_quota_reject_map`, which is dispatched to segments by this function.